index.html

```diff
@@ -174,11 +178,12 @@ <h3 class="generic section-headline">I'm always interested in hearing about new
                         <textarea name="msg" id="msg" maxlength="500" class="user-msg" required placeholder="Enter text here"></textarea>
                     </li>
                     <li>
-                        <button type="submit" class="btn">Get in touch</button>
+                        <button type="submit" class="btn btn-contact-form">Get in touch</button>
                     </li>
                 </ul>
                    
             </form>
+            </div>
             <ul class="contact-info contact-for-contact-info">
```

Comment:
- [ ] In the **Desktop** version, there is a borderline separating the icon from the contact form section. Kindly add border above icon.


- Your website 
![Screenshot from 2022-03-25 00-27-26](https://user-images.githubusercontent.com/26767383/159991204-38c9c063-ff79-4ba6-9ca7-c12ad4ec5dd0.png)

- Original design
![Screenshot from 2022-03-25 00-33-20](https://user-images.githubusercontent.com/26767383/159991458-12711f79-c355-44c0-8fe6-225352588a58.png)